### DISCUSSION TITLE
Look into adding to Microsoft Edge Store

### DISCUSSION THREAD

**Discussion by GrahamSH-LLK** [ScratchAddons/ScratchAddons#352] [Discussion]
The Microsoft Edge store supports webextensions. We should look into uploading there.

**Comment by WorldLanguages** [comment]
I'd prefer not having to upload the extension to more than one Chrom(eium) store, tbh. This could potentially be Opera, Brave (someday) and Edge, so 5 stores in total :c

**Comment by GrahamSH-LLK** [comment]
Ok.
